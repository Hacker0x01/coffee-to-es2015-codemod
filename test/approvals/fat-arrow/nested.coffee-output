var a;

a = (function(_this) {
  return function() {
    var b;
    console.info(_this);
    return b = function() {
      return console.info(_this);
    };
  };
})(this);
