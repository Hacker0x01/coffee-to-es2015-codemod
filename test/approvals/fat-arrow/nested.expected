var a;

a = () => {
  var b;
  console.info(this);
  return b = () => {
    return console.info(this);
  };
};
